api/v2/checluster_types.go

```diff
@@ -442,11 +442,11 @@ type WorkspaceStorage struct {
 }
 
 type PersistentHomeConfig struct {
-    // Determines whether the `/home/user/` directory in workspaces should persist between
-    // workspace shutdown and startup.
-    // Must be used with the 'per-user' or 'per-workspace' PVC strategy in order to take effect.
-    // Disabled by default.
-    Enabled *bool `json:"enabled,omitempty"`
+	// Determines whether the `/home/user/` directory in workspaces should persist between
```

Comment:
@tolusha is this formatting problematic?

Reply:
That's not a problem.
Could you remove references to `/home/user` from the comments in checluster_types.go and regenerate dev resources?

Reply:
Will do 👍